frontend/apps/web/app/(mgmt)/[account]/new/transformer/page.tsx

```diff
@@ -226,12 +226,12 @@ export default function NewTransformer(): ReactElement {
                     </SelectContent>
                   </Select>
                 </FormControl>
-                <FormMessage />
+                {/* <FormMessage />  we don't have any validation on this field so there's nothing to render*/}
```

Comment:
Nit: Hm, does this change the rendering? If so, then it kinda seems like `<Formmessage />` should be updated to handle that better. 
That way if this ever gets validation in the future it isn't missing its message.

Reply:
it does ever so slightly. I added it back in and then just updated the padding on the component underneath it so it's not awkwardly padded.